.github/CONTRIBUTING.MD

```diff
@@ -1,60 +1,60 @@
 # Contributing to Towny
-   
-    
-## Contributing Code    
-Code contributions are welcome and are made via GitHub's Pull Request. 
-You must fork the Towny repository (this) and then open it via your IDE.   
-You can ask for any information/help on your idea/contribution in the [Towny Discord](https://discord.gg/gnpVs5m).    
-
-* Please make sure to read our [License](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md). If you do not agree with the terms and conditions of our license, please do not contribute any code.
-* By making a pull request, you represent that you have the right to waive copyright and related rights to your contribution, and agree that all copyright and related rights in your contributions are waived, and you acknowledge that the TownyAdvanced organization has the copyright to use and modify your contribution under our [License](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md) for perpetuity.
-
-# Pull & Code Requirements    
-## Pull-Request Requirements
-1. Keep your commit message(s) self-explanitory, description should be a brief explanation and include an issue reference (if applicable.).   
-2. Pull Requests should be simple and straight-forward. One pull request for one feature. Changed Files should only show a difference where code has been removed or added, no re-formatting of entire files.   
-3. Test everything you've touched before pull-requesting. (Exception being language files, we don't expect you to check all lang strings.)    
-
-## General Code Requirements
-> With the inclusion of the Maven build system, Towny also makes use of a [EditorConfig](https://editorconfig.org) file. IDEs that support it should automatically follow parts of this section.
+Thanks for deciding to contribute to Towny! We really appreciate contributions, but to keep everything running smoothly and get contributions merged in a timely manner, we need to enforce some guidelines. :shipit:
+
+Contributions are welcomed via GitHub Pull Requests.
+You must fork this repository and then open it via your IDE.   
+You can always ask for help on your contribution in the [Towny Discord](https://discord.gg/gnpVs5m).    
+
+## Before contributing
+- Make sure to read the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md).
+- By opening an pull request, you:
+    - represent that you have the right to waive your changes' copyright and any related rights.
+    - acknowledge that the TownyAdvanced organization is allowed to use or modify your changes in any way it sees fit and in accordance with the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md)
```

Comment:
This has to be redone so that there is no change from the original wording.

Reply:
Understood!